.github/workflows/php-compatibility.yml

```diff
@@ -18,8 +18,8 @@ on:
       - '5.[5-9]'
       - '[6-9].[0-9]'
     paths:
-      # This workflow only scans PHP files.
-      - '**.php'
+      # This workflow only scans source PHP files.
+      - 'src/**.php'
```

Comment:
I think we should still run these on all files, tests and sample files still need to be compatible with the versions of PHP that WordPress supports.

Reply:
It only checks `src` files at the moment, so tests and config sample files are missed anyway: https://github.com/WordPress/wordpress-develop/blob/94cab031c85eaf86e5e6975421359b0be3b994b7/phpcompat.xml.dist#L40-L41